src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -108,6 +124,11 @@ public InstanceConfiguration(String namePrefix,
                                  String bootDiskSourceImageName,
                                  String bootDiskSourceImageProject,
                                  String bootDiskSizeGbStr,
+                                 boolean windows,
```

Comment:
The parameter list is long enough that it probably makes sense to switch to using a Builder for this class.